app/templates/events/view.hbs

```diff
@@ -1,6 +1,33 @@
 {{#unless (includes session.currentRouteName 'events.view.edit')}}
   {{!-- Tabs for the event view page will come here --}}
-  {{outlet}}
+  <div class="row">
+    <h3 class="ui left floated header">{{t 'Fossasia Test'}}</h3>
+    <div class="ui buttons right floated">
+      <button class="ui blue basic button">{{t 'Preview'}}</button>
```

Comment:
Follow a button grouping similar to:
<img width="325" alt="screen shot 2017-05-25 at 11 36 21 pm" src="https://cloud.githubusercontent.com/assets/2404372/26463839/494659e4-41a3-11e7-8152-1a6d1bb4f1ce.png">

Also, let all the other buttons (except delete) be `ui button` and not `ui basic blue button`